app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>This is users page index</h1>
\ No newline at end of file
+<div class="wrapper-1">
+  
+  <% @users.each do |user| %>
+    <li class="item">  
+       <img class="photo" src="<%= user.photo%>" alt="user photo">
+       <div class= "container"> 
+          <div class="name"> <%= user.name %> </div>
+          <div class="post-num"> number of posts: <%= user.posts_counter %></div>
+       </div>
+    </li>
```

Comment:
- [ ] It is required according to the project requirement that when a user name is clicked, it should redirect to the user's show page. This is not so in your app. Kindly look into it and fix.

[user-link.webm](https://github.com/edahigure/blog-app-2/assets/26597432/0ddca28d-f839-45c3-8a03-53d4cad7b82a)